nest-cli.json

```diff
@@ -3,6 +3,12 @@
   "collection": "@nestjs/schematics",
   "sourceRoot": "src",
   "compilerOptions": {
-    "deleteOutDir": true
+    "deleteOutDir": true,
+    "assets": [
```

Comment:
This tells the compiler to copy the contents of the src/assets folder into "dist" when the app is built. That puts all the images etc. into the folder where the compiled Javascript goes. Anything you want accessible via the web should be copied into dist.